frontend/packages/ux-editor-v3/src/containers/DesignView/PageAccordion/NavigationMenu/InputPopover/InputPopover.tsx

```diff
@@ -68,18 +69,16 @@ export const InputPopover = ({
 
   return (
     <StudioPopover open={isEditDialogOpen} onClose={handleClose}>
-      <StudioPopover.Trigger asChild>
-        <DropdownMenu.Item
-          onClick={() => setIsEditDialogOpen(true)}
-          id='edit-page-button'
-          disabled={disabled}
-          ref={newNameRef}
-          aria-expanded={isEditDialogOpen}
-        >
-          <PencilIcon />
-          {t('ux_editor.page_menu_edit')}
-        </DropdownMenu.Item>
-      </StudioPopover.Trigger>
+      <DropdownMenu.Item
+        onClick={() => setIsEditDialogOpen(true)}
+        id='edit-page-button'
```

Comment:
Not sure if it is my IDE that performs poor when searching, but I can ffind any usages of this id either 🤷